sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubMetricsProvider.cs

```diff
@@ -90,6 +90,7 @@ public async Task<EventHubsTriggerMetrics> GetMetricsAsync()
         private EventHubsTriggerMetrics CreateTriggerMetrics(List<PartitionProperties> partitionRuntimeInfo, EventProcessorCheckpoint[] checkpoints, bool alwaysLog = false)
         {
             long totalUnprocessedEventCount = 0;
+            Dictionary<string, long?> checkpointSequences = new Dictionary<string, long?>();
```

Comment:
Just for my own understanding - when would there not be a sequence number? Is this for legacy checkpoints?

Reply:
The official packages have always written one, but it's not required data.  Customer-provided checkpoint stores or manually manipulated data may not have a sequence number so long as they provided an offset.